gobblin-salesforce/src/main/java/org/apache/gobblin/salesforce/ResultChainingIterator.java

```diff
@@ -61,17 +61,17 @@ public boolean hasNext() {
 
   @Override
   public JsonElement next() {
+    if (!iter.hasNext()) {
```

Comment:
`if (!iter.hasNext()) {… }` meant to print out total info.
if iter.hasNext == false, this function won’t be called.
You can just remove the `return null` from original code.